casper/src/test/scala/coop/rchain/casper/MultiParentCasperMergeSpec.scala

```diff
@@ -40,14 +40,10 @@ class MultiParentCasperMergeSpec
         )
         block0 <- nodes(0).addBlock(deploys(0))
         block1 <- nodes(1).addBlock(deploys(1))
-        _      <- nodes(0).receive()
-        _      <- nodes(1).receive()
-        _      <- nodes(0).receive()
-        _      <- nodes(1).receive()
+        _      <- nodes.toList.traverse_(_.receive())
```

Comment:
The new code is not equivalent to the old one but looking at the test I guess 4 recieves are redundant anyway.

Reply:
Sure `.replicateA(2)`